tmail-backend/mailbox/encrypted/cassandra/src/main/java/com/linagora/tmail/encrypted/cassandra/table/CassandraEncryptedEmailStoreModule.scala

```diff
@@ -0,0 +1,36 @@
+package com.linagora.tmail.encrypted.cassandra.table
+
+import com.datastax.driver.core.DataType.{cboolean, cint, text, uuid}
+import com.datastax.driver.core.schemabuilder.{Create, SchemaBuilder}
+import com.datastax.driver.core.schemabuilder.Create.Options
+import com.datastax.driver.core.schemabuilder.SchemaBuilder.KeyCaching.ALL
+import com.datastax.driver.core.schemabuilder.SchemaBuilder.rows
+import org.apache.james.backends.cassandra.components.CassandraModule
+import org.apache.james.backends.cassandra.utils.CassandraConstants.DEFAULT_CACHED_ROW_PER_PARTITION
+
+object CassandraEncryptedEmailStoreModule {
+  val EMAIL_CONTENT_MODULE: CassandraModule = CassandraModule.table(EncryptedEmailTable.TABLE_NAME)
+    .comment("Holds content of encrypted email")
+    .options((options: Options) => options
+      .caching(ALL, rows(DEFAULT_CACHED_ROW_PER_PARTITION))
+      .compressionOptions(SchemaBuilder.lz4.withChunkLengthInKb(8)))
+    .statement((statement: Create) => statement
+      .addPartitionKey(EncryptedEmailTable.MESSAGE_ID, uuid)
+      .addColumn(EncryptedEmailTable.ENCRYPTED_PREVIEW, text())
+      .addColumn(EncryptedEmailTable.ENCRYPTED_HTML, text())
+      .addColumn(EncryptedEmailTable.HAS_ATTACHMENT, cboolean())
+      .addColumn(EncryptedEmailTable.ENCRYPTED_ATTACHMENT_METADATA, text()))
```

Comment:
Likeky `frozen(list(text()))` as a type here:

 - We get several attachement per entry here
 - Frozen as once inserted we will not modify the collection.

Reply:
Nice freature, Thanks

Reply:
`ENCRYPTED_ATTACHMENT_METADATA` is String (not List[String]), So I think we don't need `frozenList` here

Reply:
Yes I remember now it is an encrypted JSON